Source/Atomic/Scene/Node.h

```diff
@@ -495,7 +495,7 @@ class ATOMIC_API Node : public Animatable
     void RemoveChild(Vector<SharedPtr<Node> >::Iterator i);
     /// Return child nodes recursively.
     void GetChildrenRecursive(PODVector<Node*>& dest) const;
-    /// Return child nodes with a specific component recursively.
+	/// Return child nodes with a specific component recursively.
     void GetChildrenWithComponentRecursive(PODVector<Node*>& dest, StringHash type) const;

```

Comment:
Yeah, as actually doing a code review here should probably flag it :)   It is pretty easy to spot these when submitting a PR.  

You don't have to recreate the PR, if you commit to the PR branch the changes will show up in the PR